lib/a11yTestHelper.js

```diff
@@ -0,0 +1,68 @@
+// Modified from: https://gist.github.com/marcysutton/835c2fd90fadb631414b39523d5bda61
+import { findDOMNode } from 'react-dom';
+import { mount } from 'enzyme';
+import axeCore from 'axe-core';
+
+const a11yTestHelper = {};
+
+/**
+ * Test a component with Enzyme.
+ *
+ * @param {any} app - Your app reference.
+ * @param {object} config (optional) - An aXe config object to enable/disable rules. See
+ * [axe.a11yCheck](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter).
+ * @param {function} callback - A callback function to execute when aXe returns.
+ */
+a11yTestHelper.testComponent = (app, config, callback) => {
```

Comment:
Hey @emily-plummer these utils are looking fantastic! This makes it super easy to ensure `A11Y` :raised_hands: 

If we converted these helpers to utilize promises I think the integration with `Jest` would be a little cleaner.

Would be nice to set the default argument for config I think too:

```js
a11yTestHelper.testComponent = (app, config) => {/* test a11y! */};
// setting default arg with es6 syntax
a11yTestHelper.testComponent = (app, config = []) => {/* test a11y! */};
```

Useing `done` for async Jest tests is something that might to go away (I've found it to be flaky as well). If you return a promise in a Jest test, the test will wait (here's some docs on the details: https://facebook.github.io/jest/docs/tutorial-async.html).

An example coversion for the `Video` component if `a11yTestHelper.testComponent` returned a promise

```js
describe('Video', () => {
   it('should pass accessibility audit', (done) => {
     a11yTestHelper.testComponent(
       <Video
         sources={[
           {
             src: '<some src>',
             type: 'mp4',
           },
         ]}
       />, [], (results) => {
         expect(results.violations.length).toBe(0);
         done();
       });
   });
 });
```
```js
describe('Video', () => {
   it('should pass accessibility audit', () => {
     return a11yTestHelper.testComponent(
       <Video
         sources={[
           {
             src: '<some src>',
             type: 'mp4',
           },
         ]}
       />)
          .then((results) => {
              expect(results.violations.length).toBe(0);
          });
   });
 });
```